app/models/groups_metadata_teams_committees.rb

```diff
@@ -3,6 +3,12 @@
 class GroupsMetadataTeamsCommittees < ApplicationRecord
   has_one :user_group, as: :metadata
 
+  enum :preferred_contact_mode, {
+    no_public_way: "no_public_way",
```

Comment:
Naming choice here feels a bit odd. What about `private`?

Reply:
Changed to no_contact as per discussion.